lib/mobility/plugins/attribute_methods.rb

```diff
@@ -34,5 +34,7 @@ def include_attribute_methods_module(model_class, *attribute_names)
         model_class.include module_builder.new(*attribute_names)
       end
     end
+
+    register_plugin(:attribute_methods, AttributeMethods)
```

Comment:
Thinking this module should really go in v1.0, it's very AR/Rails specific. Maybe a separate gem or something, or just a note in the wiki with some copy-paste code since it's so small anyway.